src/transformers/models/auto/tokenization_auto.py

```diff
@@ -184,6 +184,10 @@
                 "fastspeech2_conformer",
                 ("FastSpeech2ConformerTokenizer" if is_g2p_en_available() else None, None),
             ),
+            (
+                "fastspeech2_conformer_with_hifigan",
+                ("FastSpeech2ConformerTokenizer" if is_g2p_en_available() else None, None),
+            ),
```

Comment:
Otherwise, the pipeline don't recognize a tokenizer class and thus set it to `None`